### PR TITLE
cloud: TiDB Serverless supports PITR

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

TiDB Serverless supports PITR
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by pingyu — APPROVED** [review]
LGTM~

**Review by ran-huang — APPROVED** [review]

File: tidb-cloud/backup-and-restore-serverless.md [hunk 1]
```diff
@@ -56,9 +55,33 @@ TiDB Serverless only supports in-place restoration. To restore your TiDB Serverl
...
+    2. Select the backup snapshot you want to restore to.
+
+    </div>
+    <div label="Point-in-Time Restore">
+
+    > **Note:**
+    >
+    > The **Point-in-Time Restore** feature is currently in beta.
+
+    To restore from any point in time within the backup retention, take the following steps:
```
Comment (pingyu) [inline tidb-cloud/backup-and-restore-serverless.md:77]: Within 90 days, and not before the cluster creation time.

File: tidb-cloud/backup-and-restore-serverless.md [hunk 2]
```diff
@@ -56,9 +55,33 @@ TiDB Serverless only supports in-place restoration. To restore your TiDB Serverl
...
+    <SimpleTab>
+    <div label="Basic Snapshot Restore">
+
+    To restore from a selected backup snapshot, take the following steps:
+
+    1. Click **Basic Snapshot Restore**.
+    2. Select the backup snapshot you want to restore to.
+
+    </div>
+    <div label="Point-in-Time Restore">
```
[the remark this answers is not in the document]
Comment (Oreoxmt) [reply tidb-cloud/backup-and-restore-serverless.md:71]: @pingyu PTAL

File: tidb-cloud/backup-and-restore-serverless.md [hunk 3]
```diff
@@ -56,9 +55,35 @@ TiDB Serverless only supports in-place restoration. To restore your TiDB Serverl
...
+
+    This feature lets you restore a cluster to a specific state from any time within the last 90 days.
+
+    > **Note:**
+    >
+    > The **Point-in-Time Restore** feature is currently in beta.
+
+    To restore from any point in time within the backup retention, take the following steps:
```
Comment (Oreoxmt) [inline tidb-cloud/backup-and-restore-serverless.md:79]:
```suggestion
    To restore from a specific point in time, take the following steps:
```

**Comment by pingyu** [comment]
LGTM~

**Pull request merged**